### PR TITLE
SEO-176062-xamarin-UG-redirects-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: Xamarin/DataGrid/Paging.md [hunk 1]
```diff
@@ -390,4 +390,4 @@ N> You can refer to our [Xamarin DataGrid](https://www.syncfusion.com/xamarin-ui
 
 ## See also
 
-[How to have different page size in SfDataPager with SfDataGrid when the device is in horizontal and vertical orientation mode](https://www.syncfusion.com/kb/9299)
+[How to have different page size in SfDataPager with SfDataGrid when the device is in horizontal and vertical orientation mode](https://support.syncfusion.com/kb/article/7968/how-to-have-different-page-size-in-sfdatapager-with-sfdatagrid-when-the-device-is-in?isInternalRefresh=False)
```
Comment (Yvone-Atieno) [inline Xamarin/DataGrid/Paging.md:393]: Check KB links

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/DataGrid/Paging.md (hunk 1 above)
Comment (Edithabich) [reply Xamarin/DataGrid/Paging.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**